### DISCUSSION TITLE
Prospective Student (GSoC 2023)

### DISCUSSION THREAD

**Discussion by daijingz** [wagtail/gsoc#17] [General]
Hi everyone,

My name is Jingze Dai, and I am a fourth-year computer science student at McMaster University, Canada. My proficiency includes python, Java, C++ and C. This is my first-time official participation in GSOC. I spent a lot of time learning programming languages, professional libraries, and tools.

I am willing to contribute to WagTail. However, I have some contribution hesitations: I cannot accept the situation when I give code contributions but the coordinator doesn’t allow me to get in. I am quite uncertain. This is my last year of being an undergraduate student, so I believe I cannot waste time and lose this opportunity.

My plan is, at first, I contact the coordinator so that I can discuss my concerns with him. If the coordinator allows me to contribute codes and join GSoC 2023, I will have 3 – 5 contributions until March 24th, the registration date.

Once it is determined, Jingze will spend a lot of time contributing codes. Everything is easy to discuss through emails.

(If you are the coordinator, can you give me an opportunity to join in GSoC 2023?)

Thank you
Jingze
[david1147062956@gmail.com](mailto:david1147062956@gmail.com)

**Comment by thibaudcolas** [comment]
Welcome @daijingz! Please make sure to read [our contributors guide](https://github.com/wagtail/gsoc/blob/main/contributor-guide.md) which explains what Google and us expect (for example where to introduce yourself and how).

We’re not expecting nor asking people to contribute code by the registration date, and we don’t let people "in" or "out" at this stage.
